controllers/storagecluster/storageclient.go

```diff
@@ -21,6 +25,10 @@ const (
 	ocsClientConfigMapName             = "ocs-client-operator-config"
 	manageNoobaaSubKey                 = "manageNoobaaSubscription"
 	useHostNetworkForCsiControllersKey = "useHostNetworkForCsiControllers"
+	// cephNetworkAnnotationKey is the annotation key used to store network details used by ceph
+	cniNetworksAnnotationKey = "k8s.v1.cni.cncf.io/networks"
+	// cephPublicNetworkType is the type of network used by Ceph for public communication
+	cephPublicNetworkType = "public"
```

Comment:
We we get rid of this const? it is being used in a very specific context and the value is self explanatory in that context. Better to just inline this